test/e2e/apimachinery/garbage_collector.go

```diff
@@ -328,15 +330,23 @@ func newCronJob(name, schedule string) *batchv1beta1.CronJob {
 	}
 }
 
+// getUniqLabel returns a UniqLabel based on labeLkey and labelvalue.
+func getUniqLabel(labelkey, labelvalue string) map[string]string {
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	suffix := strconv.Itoa(r.Int() % 10000)
```

Comment:
count up with an atomic integer perhaps?

Reply:
Sure,I have re push it,Thanks